SPIE2016-9913-16/9913-16.tex

```diff
@@ -0,0 +1,380 @@
+The framework includes a Python-based configuration language that has default values, overrides from code as well as the end user, and hooks for validation of configured values.
+
+\end{description}
+
+The LSST code base currently contains components that are not of short-term interest for Astropy integration but might become so in the longer run given related work by Astropy contributors in some of these areas\cite{2016-PyAstro-Turner,2016_O12.3_adassxxv}.
+These include the Data Butler and the Task framework.
+In terms of overlap with Astropy and affiliate packages the \texttt{afw} and \texttt{meas} packages are most relevant for immediate investigation.
+
+The algorithms in the LSST codebase are not yet at the level of scientific quality ultimately needed for LSST, but they are already being used to process data from the Hyper Suprime-Cam Survey on the Subaru telescope\cite{2012SPIE.8446E..0ZM}, one of the major precursors surveys for LSST, and generally represent the current state-of-the-art.
+
+
+\section{Astropy}
+
+Python has become the dominant programming language for astronomy\cite{2000ASPC..216...59G,2006ASPC..351..343H,2006ASPC..351..497K,2011ASPC..442..425G,2012SPIE.8451E..02J} and is now the default choice for new projects and for teaching.
+The Astropy\cite{2013A&A...558A..33A} project grew out of a support mailing list in 2011 to become a very successful hub for the development of Python packages for astronomy.
+At the time of writing Astropy consists of approximately 105,000 lines of Python code.
+Astropy has two key approaches to drive adoption.
+The first is to place generic astronomy code into the \emph{core} package that is brought in via \texttt{import astropy} to provide a solid, well-tested base for astronomy software development; the second is to support and encourage the concept of \emph{affiliated} packages, which use the \texttt{astropy} package and follow the same philosophy on distribution and openness.
+Together these aim to provide a consistent environment of highly-usable, well-documented, easy-to-install code for astronomers trying to use Python in their research.
+
+\section{Synergies}
+
+The LSST project is constrained by the requirements of the funding agencies to deliver a data management system on schedule and on budget.
+Involvement with Astropy is justifiable if it helps the project improve schedule or performance or if it improves community usability while not impacting deliverables.
+It is also possible that making it easier for the community to adopt LSST algorithms can improve robustness of the code and find bugs earlier.
+The community will be encouraged to try out the LSST software sooner if it integrates well with Astropy.
+This is likely to significantly enhance utilization of the compute resources provided at LSST data access centers, the Level 3 processing system, during LSST operations resulting in a system that is more scientifically productive.
```

Comment:
Maybe something simpler like:

"This will likely facilitate adoption of the Level 3 processing system during LSST operations, resulting in a system that is more scientifically productive."